src/pybind/mgr/snap_schedule/module.py

```diff
@@ -70,7 +70,7 @@ def snap_schedule_get(self,
             return -errno.EINVAL, '', f"no such filesystem: {use_fs}"
         try:
             ret_scheds = self.client.get_snap_schedules(use_fs, path)
-        except CephfsConnectionException as e:
+        except (sqlite3.OperationalError, CephfsConnectionException) as e:
```

Comment:
ceph-mgr modules are _not_ cleaned up on termination. This means that they are kind of interrupted mid-way during execution. Maybe that's what causes this exception. If this isn't a problem with the tests, I recommend that we catch a general exception (`Exception`), log and return the error string to the user.

Reply:
@vshankar do you want the change in this file as a separate PR ? (since this is not a *qa* specific change)

Reply:
that makes sense.